ui/src/app/model/workflow.run.model.ts

```diff
@@ -39,6 +41,20 @@ export class WorkflowRun {
     // Useful for UI
     duration: string;
     force_update: boolean;
+
+    static fromEventRunWorkflow(event: Event): WorkflowRun {
+        let wr = new WorkflowRun();
+        wr.id = event.payload['ID'];
```

Comment:
Just for information, why don't we use the json annotation in go to not use the real go fields name ?

Reply:
Event use go-dump